code/modules/research/designs/stock_parts_designs.dm

```diff
@@ -32,6 +32,7 @@
 	materials = list(MAT_METAL = 100, MAT_GLASS = 100)
 	build_path = /obj/item/weapon/stock_parts/capacitor
 	category = list("Stock Parts","Machinery","initial")
+	lathe_time_factor = 0.2
```

Comment:
Recommendation: Make `/datum/design/stock_parts/` as a parent for all of the stock parts, so you only need to set `lathe_time_factor = 0.2` once, and should you need to change it, you only need it in one place. (copy pasta rule)